apps/bc-wallet-showcase-creator/app/[locale]/(public)/[tenant]/login/page.tsx

```diff
@@ -0,0 +1,14 @@
+import { LoginForm } from "@/components/login-form"
+import { auth } from "@/auth"
+import { redirect } from "next/navigation"
+
+export default async function LoginPage(promise: { params: Promise<{ tenantId?: string; locale: string }> }) {
```

Comment:
```suggestion
export default async function LoginPage(promise: { params: Promise<{ tenantId: string; locale: string }> }) {
```
this is not optional